packages/cli/src/aws/init.ts

```diff
@@ -270,7 +272,7 @@ export async function initStackCommand(): Promise<void> {
     await writeParameters(config.region, `/medplum/${config.name}/`, serverParams);
   } else {
     const serverConfigFileName = configFileName.replace('.json', '.server.json');
-    writeConfig(serverConfigFileName, serverParams);
+    writeConfig(config, config.name + '.server');
```

Comment:
This is incorrect -- `serverParams` are the values for AWS Parameter Store, which is separate than the infra config.

It is totally valid critique that that distinction is bad, but that's the way it is for now :/